app/models/concerns/push_updates.rb

```diff
@@ -8,14 +8,22 @@ module PushUpdates
   end
 
   def update_client_store(operation)
+    type  = self.class.name
```

Comment:
I wonder if it would be best to have a separate OrderPushUpdate class here. Perhaps inheriting from PushUpdateBase. Let's talk this one through and come up with a plan.

Reply:
Sure, It would be better as later we might include pusUpdates for other models also.